gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Just double checking: This file is automatically generated, right?

Reply:
Yes, `gradlew` and `gradlew.bat` are automatically generated by Gradle's [wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) task. You will see subtle changes over the course of Gradle versions as they improve these generated scripts.